pkg/buffer/buffer.go

```diff
@@ -22,6 +24,8 @@ const (
 	reportDelta = 1e9
 )
 
+var defaultLogger, debugLogger logr.Logger
```

Comment:
```suggestion
var Logger logr.Logger = new(logr.DiscardLogger)
```

And update the binaries/main packages to explictly set logging instances

```
func setLoggers(/* some parameters via cmd line or similar */) {
    buffer.Logger = log.New()
    ...
}
```

Reply:
Updated. But only for sfu, because in cmd/ is no explicit buffer package using